indexer/ai/src/main/java/com/google/cloudsearch/ai/CloudStorageHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloudsearch.ai;
+
+public class CloudStorageHandler {
```

Comment:
It would be better to rename this file to something like: Util.java. All other common utility functions can be added in this file as well.

Reply:
I actually wanted to keep at least 1 dedicated class for Cloud Storage related things. Like we may need to add functions to check size etc, so it would be good to have a class for all cloud storage related things. If there are other utilities required then I can add that to a new class Util.java.